bentoml/sklearn.py

```diff
@@ -130,15 +130,11 @@ def __init__(
         batch_options: t.Optional[t.Dict[str, t.Any]],
         model_store: "ModelStore" = Provide[BentoMLContainer.model_store],
     ):
+        tag = model_store.get(tag).tag
```

Comment:
let's put a hold on this PR and redo this change after https://github.com/bentoml/BentoML/pull/2108/ is merged

Reply:
sure